src/main.scss

```diff
@@ -1,11 +1,13 @@
-.section.tags,
+#navigation.section,
+.section.tags, // legacy code pre-smart tags release
 navigation {
   flex: none !important;
   width: 120px !important;
   transition: width 0.25s;
 }
 
-.section.tags:hover,
+#navigation.section:hover,
+.section.tags:hover, // legacy code pre-smart tags release
```

Comment:
@amanharwara sorry for bumping you again, I'm not familiar with themes yet,
It's ready to merge, do I need to bump other repos or it'll be rebuilt automatically?

Reply:
You'll need to manually bump the version in the [package.json](https://github.com/standardnotes/dynamic-theme/blob/master/package.json) and then merge it. Then you can create a task for @johnny243 to create a new release.